transports/azure-storage-queues/operations-scripting.md

```diff
@@ -0,0 +1,110 @@
+
+For easier access, the connection string can be stored as an environment variable.
+
+```
+#get the connection string, 
+$connectionString=az storage account show-connection-string -n $storageAccount -g $resourceGroup --query connectionString -o tsv
+
+#set conn string as env variable
+$env:AZURE_STORAGE_CONNECTION_STRING = $connectionString
+```
+
+## Create Queues
+
+Queue creation can be done for a specific endpoint or queues shared between multiple endpoints.
+
+Note: The Azure Storage Queues Transport will fail to start if a queue name is longer than 63 characters. Refer to the [sanitization document](/transports/azure-storage-queues/sanitization.md) for more details.
+
+```
+#check if queue exists
+az storage queue exists -n "endpointname"
+
+#create a queue
+az storage queue create -n "endpointname"
+
+#create a error queue
+az storage queue create -n "error"
+```
+
+## Delete Queues
+
+```
+az storage queue delete -n "endpointname"
+```
+
+## Publish/Subscribe
+
+Azure Storage Queue transport implements the publish/subscribe (pub/sub) pattern. 
+
+Implementaion of the pub/sub pattern via Azure CLI, involves [creation of the endpoint queues](/transports/azure-storage-queues/operations-scripting.md#create-queues) (publisher and subscriber), followed by the subscription table and then the subscription entity in that table
+
+#### Create the subscription routing table
+
+In a pub/sub pattern, the transport creates a dedicated subscription routing table shared by all endpoints, which holds subscription information for each event type.
+
+```
+az storage table create -n "subscriptions"
+```
+
+#### Create a subscription
+
+When the subscriber endpoint subscribes to an event, an entity is created in the subscription routing table.
+When the publisher endpoint publishes an event, the subscription routing table is queried to find all of the subscribing endpoints.
+
+```
+az storage entity insert --entity PartitionKey=MyEvent RowKey=Subscriber  Address=subscriber Endpoint=Subscriber Topic=MyEvent --if-exists fail --table-name subscriptions
+```
+
+## Unsubscribe
+
+To unsubscribe, delete the entity from the subscriptions table
+
+```
+az storage entity delete --partition-key  MyEvent  --row-key  Subscriber  --table-name subscriptions  --if-match *
+```
+
+## Delayed delivery
+
+When delayed delivery is enabled at an endpoint, the transport creates a storage table to store the delayed messages. 
+By default, the storage table name is constructed using a naming scheme that starts with the word delays followed by SHA-1 hash of the endpoint's name.
+
+```
+#table for publisher with SHA-1 hash of 11f9578d05e6f7bb58a3cdd00107e9f4e3882671
+az storage table create -n "delays11f9578d05e6f7bb58a3cdd00107e9f4e3882671"
+
+#table for subscriber with SHA-1 hash of 17b874d289117b1353bc5080960074585aed4227
+az storage table create -n "delays17b874d289117b1353bc5080960074585aed4227" 
```

Comment:
Since message delays are not specific (or allowed) in Pub/Sub, there is no need to make the example specific to pub/sub.

```suggestion
```